packages/revocation/src/implementations/statuslistentry-verification.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  CredentialSubject,
+  StatusList2021Entry,
+  VerifiableCredential,
+} from '@ew-did-registry/credentials-interface';
+import * as pako from 'pako';
+import axios from 'axios';
+
+/**
+ * A Class to verify the StatusList2021Entry
+ */
+export class StatusListEntryVerification {
+  private verify: (vc: string, proof_options: string) => Promise<any>;
+
+  constructor(
+    callbackFunction: (vc: string, proof_options: string) => Promise<any>
```

Comment:
I think `callbackFunction` is too generic.
```suggestion
    vcProofVerificationCallback: (vc: string, proof_options: string) => Promise<any>
```